implementation.md

```diff
@@ -419,8 +429,28 @@ is used to discover the properties of the action.
 
 For details on the parameters to an action, see below.
 
+These two methods are a bit different in how they operate. Actions provided by
+a method are limited by the instances, or one globally. For instance, the SFTP
+plugin allows creating file deletion commands. There's only one way to delete a
+file, so it is in a method per instance. The `Definer` method allows
+dynamically creating actions. For instance, the Vidarr plugin will scan the
+available workflows on a remote instance and create on action definition for
+each workflow type. The `Definer` allows creating custom parameters, so in the
+workflow case, some parameters are baked into the Vidarr submit action, but
```

Comment:
So at this point some piece of code has scanned a remote instance and created Actions for each workflow type. In this context I am not sure what 'in the workflow case' means, because it sounds like we only have workflows to work with